docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+
+```c
+#include <webrtc.h>
+```
+
+<a name="media_source"></a>
+## Managing Media Sources
+
+You can add media sources to a webrtc handle. Once you get source id of the media source, you can manage various functions of the media source with the source id.
+
+1. To add a media source to the webrtc handle, use `webrtc_add_media_source()` function before calling `webrtc_start()` function:
+
+    ```c
+    int ret;
+    webrtc_h webrtc;
+    unsigned int a_src_id;
+    unsigned int v_src_id;
+
+    ret = webrtc_create(&webrtc);
+    ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_AUDIOTEST, &a_src_id);
+    ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_VIDEOTEST, &v_src_id);
+    ...
+    ret = webrtc_start(webrtc);
+    ```
+
+2. To set or get the direction of the media source, use `webrtc_media_source_set_transceiver_direction()` or `webrtc_media_source_get_transceiver_direction()` function:
+   
+    Default transceiver direction of a media source is `WEBRTC_TRANSCEIVER_DIRECTION_SENDRECV`.
+
+    In the following example code, it tries to change the direction to `WEBRTC_TRANSCEIVER_DIRECTION_SENDONLY` before it starts.
+
+    ```c
+    ret = webrtc_media_source_set_transceiver_direction(webrtc, v_src_id, WEBRTC_MEDIA_TYPE_VIDEO, WEBRTC_TRANSCEIVER_DIRECTION_SENDONLY);
+    ...
+    ret = webrtc_start(webrtc);
+    ```
+
+3. To pause a media source, use the `webrtc_media_source_set_pause()` function:
+
+    ```c
+    ret = webrtc_media_source_set_pause(webrtc, a_src_id, WEBRTC_MEDIA_TYPE_AUDIO, true);
+    ```
+    > **Note**
+    >
+    > Pausing a media source means it does not send the data to a remote peer. Pause or resume of a media source is also possible in `WEBRTC_STATE_PLAYING`.
+
+4. To mute a media source, use the `webrtc_media_source_set_mute()` function:
+
+    ```c
+    ret = webrtc_media_source_set_mute(webrtc, a_src_id, WEBRTC_MEDIA_TYPE_AUDIO, true);
+    ret = webrtc_media_source_set_mute(webrtc, v_src_id, WEBRTC_MEDIA_TYPE_VIDEO, true);
+    ```
+    > **Note**
+    >
+    > Muting a media source means it sends black video frames or silent audio frames to a remote peer. Mute or un-mute of a media source is also possible in `WEBRTC_STATE_PLAYING`.
+    > Some types of media sources do not support this functionality. For example, `WEBRTC_MEDIA_SOURCE_TYPE_FILE` and `WEBRTC_MEDIA_SOURCE_TYPE_MEDIA_PACKET`.
+
+4. To set or get the video resolution to a media source, use the `webrtc_media_source_set_video_resolution()` or `webrtc_media_source_get_video_resolution()` function:
+
+    ```c
+    ret = webrtc_media_source_set_video_resolution(webrtc, v_src_id, 640, 480);
+    ```
+    > **Note**
```

Comment:
For alerts, please use alert formatting.
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning
```suggestion
    > [!NOTE]
```